packages/contracts/test/src/CredbullFixedYieldVaultTest.t.sol

```diff
@@ -2,46 +2,51 @@
 
 pragma solidity ^0.8.20;
 
+import { ERC20 } from "@openzeppelin/contracts/token/ERC20/ERC20.sol";
 import { IAccessControl } from "@openzeppelin/contracts/access/IAccessControl.sol";
 import { Pausable } from "@openzeppelin/contracts/utils/Pausable.sol";
 
-import { Test } from "forge-std/Test.sol";
-
-import { HelperConfig } from "@script/HelperConfig.s.sol";
-import { DeployVaultFactory } from "@script/DeployVaultFactory.s.sol";
-
 import { CredbullFixedYieldVault } from "@credbull/CredbullFixedYieldVault.sol";
 import { CredbullWhiteListProvider } from "@credbull/CredbullWhiteListProvider.sol";
 import { WhiteListProvider } from "@credbull/provider/whiteList/WhiteListProvider.sol";
 import { WhiteListPlugin } from "@credbull/plugin/WhiteListPlugin.sol";
 
-import { ParamsFactory } from "@test/test/vault/utils/ParamsFactory.t.sol";
+import { AltParamsFactory } from "@test/test/vault/utils/AltParamsFactory.t.sol";
 import { SimpleUSDC } from "@test/test/token/SimpleUSDC.t.sol";
+import { TestDeployVaults, TestDeployVaultsSupport } from "@test/test/deployer/TestDeployVaults.t.sol";
+import { VaultsSupportConfiguredTest } from "@test/test/deployer/ConfiguredTest.t.sol";
+
+contract CredbullFixedYieldVaultTest is VaultsSupportConfiguredTest {
+    uint256 private constant INITIAL_BALANCE = 1000;
+    uint256 private constant ADDITIONAL_PRECISION = 1e12;
+
+    address private alice = makeAddr("alice");
+    address private bob = makeAddr("bob");
 
-contract CredbullFixedYieldVaultTest is Test {
+    TestDeployVaults private vaultsDeployer;
+    TestDeployVaultsSupport private vaultsSupportDeployer;
     CredbullFixedYieldVault private vault;
-    HelperConfig private helperConfig;
-    DeployVaultFactory private deployer;
     CredbullWhiteListProvider private whiteListProvider;
 
     CredbullFixedYieldVault.FixedYieldVaultParams private params;
 
-    address private alice = makeAddr("alice");
-    address private bob = makeAddr("bob");
-
     uint256 private precision;
-    uint256 private constant INITIAL_BALANCE = 1000;
-    uint256 private constant ADDITIONAL_PRECISION = 1e12;
 
     function setUp() public {
-        deployer = new DeployVaultFactory();
-        (,, whiteListProvider, helperConfig) = deployer.runTest();
-        params = new ParamsFactory(helperConfig.getNetworkConfig()).createFixedYieldVaultParams();
+        vaultsDeployer = new TestDeployVaults();
+        vaultsSupportDeployer = new TestDeployVaultsSupport();
+
+        (,, whiteListProvider) = vaultsDeployer.deploy();
+        (ERC20 cbl, ERC20 usdc,) = vaultsSupportDeployer.deploy();
+
+        params = new AltParamsFactory(usdc, cbl).createFixedYieldVaultParams();
```

Comment:
The `ParamsFactory` really only needs access to the config and the deployed `CBL` and `USDC` tokens.